.travis.yml

```diff
@@ -29,8 +29,8 @@ matrix:
           name: wxGTK 2 clang Ubuntu 18.04
         - dist: bionic
           compiler: gcc
-          env: wxTOOLSET=cmake wxCMAKE_GENERATOR="Unix Makefiles"
-          name: wxGTK CMake Ubuntu 18.04
+          env: wxGTK_VERSION=3 wxTOOLSET=cmake wxCMAKE_GENERATOR="Unix Makefiles"
```

Comment:
Setting `wxGTK_VERSION` here is misleading, as it has no effect. GTK3 will be used because that is the default. Is there a way to select a GTK that is not the default?

Reply:
It is used to install the gtk3 libraries (`before_install.sh`).

CMake has the `wxBUILD_TOOLKIT` option.
For Travis you can add ` wxCMAKE_DEFINES="-DwxBUILD_TOOLKIT=gtk2"` to `.travis.yml`. 
But I was fine with using GTK3 in the Travis build.